store/src/main/java/com/nytimes/android/external/store/base/impl/ParsingStoreBuilder.java

```diff
@@ -81,13 +84,22 @@ public Raw call() throws Exception {
 
     @NonNull
     public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Func1<Raw, Parsed> parser) {
-        this.parser = parser;
+        this.parsers.clear();
+        this.parsers.add((Parser<Raw, Parsed>) parser);
         return this;
     }
 
     @NonNull
     public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Parser<Raw, Parsed> parser) {
```

Comment:
```
        Store<String> simpleStore = ParsingStoreBuilder.<String, String>builder()
                .persister(persister)
                .fetcher(fetcher)
                .parser(parser1)
                .parser(parser2)
                .open();
```

In the above example we want <String, String> Store so parser1 would have to be <String, X> and parser2 <X, String>. It would be impossible to achive with current generics.

We would need to remove generics from arguments: 

```
    @NonNull
    public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Func1 parser) {
        this.parsers.add((Parser) parser);
        return this;
    }

    @NonNull
    public ParsingStoreBuilder<Raw, Parsed> parser(final @NonNull Parser parser) {
        this.parsers.add(parser);
        return this;
    }

    @NonNull
    public ParsingStoreBuilder<Raw, Parsed> parsers(final @NonNull List<Parser> parsers) {
        this.parsers.addAll(parsers);
        return this;
    }
```